asa/ja-jp/Content/Topics/Adv_Server_Access/docs/about-asa.htm

```diff
@@ -1,272 +1,272 @@
+                                    <div id="coveo_token_url" style="Display:None"><span class="mc-variable okta-coveo-config.SearchTokenServiceUrl variable">https://support.okta.com/help/services/apexrest/PublicSearchToken?site=help</span>                                     </div>                                     <!-- End Coveo config parameters --> <!-- ************** Coveo Search bar ************************************* --> <!-- set margin and size in /Content/Resources/Scripts/coveo-resources/css/Coveo.Okta.StandaloneSearchbox.min.css to make room for replacing Flare-generated top-nav with HTML code (see below)-->                                     <div id="OLC_Coveo_Headline" class="coveo-headline-wrapper" data-mc-conditions="MultiProdPubish.SearchBar">
+                                        <div class="slds-p-vertical_small">                                             <!--Search Bar-->                                             <div class="slds-grid slds-grid_align-center">
+                                                <div id="customSelect" class="CoveoCustomSelect slds-float_right">
+                                                </div>
+                                                <div id="standaloneSearchbox">
+                                                    <div class="CoveoAnalytics" data-search-hub="OktaCommunityFullSearch">
+                                                    </div>
+                                                    <div class="CoveoTab" data-id="Help" data-caption="Help" style="display:none">
+                                                    </div>
+                                                    <div class="CoveoSearchbox" data-enable-omnibox="true" data-enable-query-suggest-addon="true">
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="replace_top_nav">
+                                        <div class="navbar">
+                                            <div class="dropdown" data-mc-conditions="MultiProdPubish.DocMenu">                                                 <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>                                                 <div class="dropdown-content">
+                                                    <div><a href="https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm">リリースノート</a>                                                     </div>
+                                                    <div><a href="https://help.okta.com/ja-jp/Content/index.htm">製品ドキュメント</a>                                                     </div>
```

Comment:
this changes we should automate, we have https://oktainc.atlassian.net/browse/OKTA-540003 for it